libraries/api-extractor/src/DocElementParser.ts

```diff
@@ -7,13 +7,13 @@ import Tokenizer from './Tokenizer';
 import ResolvedApiItem from './ResolvedApiItem';
 
 export default class DocElementParser {
-
   /**
-   * Matches only strings that contain characters for words.
-   * Any non word characters or spaces, will be present in the third entry in the match results
-   * if they exist.
+   * Used to validate the display text for an \@link tag.  The display text can contain any
+   * characters except for certain reserved AEDoc delimiters: "@", "|", "{", "}".
+   * This RegExp matches the first bad character.
```

Comment:
Nitpicky, but standardize on either "reserved" (used here) or "unsupported" (used in the error message).